imb2139.md

```diff
@@ -0,0 +1,41 @@
+# Open Source Development
+
+Name: Ivy Basseches
+
+UNI: imb2139
+
+Github: [link](https://github.com/basseches)
+
+
+## Project Proposal
+
+I'm still not totally sure what direction I want to go in for the project, so
+here are a few of my initial ideas.
+
+### LaTeX generator
+
+I use LaTeX a lot, and I've noticed that it can be very repetitive and tedious
```

Comment:
I would recommend working in the `pandoc`/`nbconvert` ecosystem, Jupyter notebook -> nice PDF is a big topic